src/jitter.rs

```diff
@@ -48,12 +48,12 @@ const MEMORY_SIZE: usize = MEMORY_BLOCKS * MEMORY_BLOCKSIZE;
 // Note: the C implementation relies on being compiled without optimizations.
 // This implementation goes through lengths to make the compiler not optimise
 // out what is technically dead code, but that does influence timing jitter.
-pub struct JitterRng {
+pub struct JitterRng<T: JitterTimer> {
```

Comment:
Turns out, this is a problem: `JitterRng` is no longer a concrete type, so we can't use it in the `EntropyRng` enum. I'm going to revert this commit: 3aa580ac4b